staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -33,6 +33,14 @@ const (
 	// StreamingProxyRedirects controls whether the apiserver should intercept (and follow)
 	// redirects from the backend (Kubelet) for streaming requests (exec/attach/port-forward).
 	StreamingProxyRedirects utilfeature.Feature = "StreamingProxyRedirects"
+
+	// owner: timstclair
+	// alpha: v1.7
+	//
+	// AdvancedAuditing enables a much more general API auditing pipeline, which includes support for
+	// pluggable output backends and an audit policy specifying how different requests should be
+	// audited.
+	AdvancedAuditing utilfeature.Feature = "AdvancedAuditing"
```

Comment:
What do we want to switch here? By default auditing is off. If you use the `--audit-log-*` flags, you will get the log plugin. What about the default level? It used to be meta data only, but not "none".

We could re-add the old `WithAudit` filter as `WithBasicAudit` and use that if the feature is disabled. Wdyt?